### DISCUSSION TITLE
is there any version that support php 8.0?

### DISCUSSION THREAD

**Discussion by Harun-or-rashid** [cjmellor/approval#20] [Q&A] [answered]
I need a similar service for my laravel application which is containing php 8.0* .
is there any verson that i can use it through my application?

**Comment by cjmellor** [comment, marked as answer]
Hi,

I've put the minimum PHP version as 8.1 and I am using 8.1 features in the package. 

If you need to use it on v8.0 I can only suggest to you to fork the repo and convert it to work with v8.0 then add it to your `composer.json`

Hope that helps.

**Reply by Harun-or-rashid** [reply]
Thanks a lot
